routes/api/team.js

```diff
@@ -93,12 +93,12 @@ module.exports = {
         );
 
         /**
-         * @api {get} /team/:id get a team's information
+         * @api {get} /team/ get a team's information
          * @apiName getTeam
          * @apiGroup Team
          * @apiVersion 0.0.8
          * 
-         * @apiParam (param) {ObjectId} id a team's unique mongoId
+        * @apiParam (body) {MongoId} MongoId of the team
```

Comment:
Invalid param name (should be `id`, not `MongoId`). Change documentation to be: {ObjectId} id The ID of the team